nucleus/core/kernel/src/test/java/org/glassfish/runnablejar/commandline/CommandLineParserTest.java

```diff
@@ -0,0 +1,64 @@
+package org.glassfish.runnablejar.commandline;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.of;
+
+class CommandLineParserTest {
+
+    private PrintStream originalPrintStream;
+
+    @BeforeEach
+    void setup() {
+        originalPrintStream = System.err;
+    }
+
+    @AfterEach
+    void restore() {
+        System.setErr(originalPrintStream);
+    }
+
+    static Stream<org.junit.jupiter.params.provider.Arguments> portValues() {
+        return Stream.of(
+                of((Object) new String[]{"--port=9090"}),
+                of((Object) new String[]{"--port", "9090"})
+        );
+    }
+
+    static Stream<org.junit.jupiter.params.provider.Arguments> verboseValues() {

```

Comment:
would be better to move this method just above the `parseVerboseOption` method, where it's used.